pkgs/by-name/ms/ms-edit/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "ms-edit";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "edit";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-5GUAHa0/7k4uVNWEjn0hd1YvkRnUk6AdxTQhw5z95BY=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-DEzjfrXSmum/GJdYanaRDKxG4+eNPWf5echLhStxcIg=";
+  # Requires nightly features
+  env.RUSTC_BOOTSTRAP = 1;
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgram = "${placeholder "out"}/bin/${finalAttrs.meta.mainProgram}";
+  versionCheckProgramArg = "--version";
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
I think (to be checked in the `buildRustPackage` wrapper), that this is automatically set.

Reply:
`rg update pkgs/build-support/rust` doesn't show anything that would indicate to me that is true.